tests/testsuite/patch.rs

```diff
@@ -928,20 +928,26 @@ fn add_patch_with_features() {
         .file("bar/src/lib.rs", r#""#)
         .build();
 
-    p.cargo("check").with_stderr_data(str![[r#"
-[WARNING] patch for `bar` uses the features mechanism. default-features and features will not take effect because the patch dependency does not support this mechanism
+    p.cargo("check")
+        .with_stderr_data(str![[r#"
+[WARNING] unused field in patch for `bar`: `features`
+[NOTE] configure `features` in the `dependencies` entry
```

Comment:
Add this to make two sentences more connected?

```suggestion
[NOTE] configure `features` in the `[dependencies]` entry instead
```

Reply:
That would take the two sentences and make them one which imo doesn't quite fit with this style of messages.

Reply:
That's fair!